xmtp_mls/src/groups/group_permissions.rs

```diff
@@ -242,8 +243,9 @@ impl PolicySet {
         Self {
             add_member_policy,
             remove_member_policy,
-            add_installation_policy: default_add_installation_policy(),
-            remove_installation_policy: default_remove_installation_policy(),
+            add_installation_policy: MembershipPolicies::allow(),
+            remove_installation_policy: MembershipPolicies::deny(),
+            update_group_name_policy: MembershipPolicies::allow() 
```

Comment:
@cameronvoell  What should this `update_group_name_policy` be default set to?

cc @neekolas